api/src/modules/impact/impact.service.ts

```diff
@@ -144,6 +144,21 @@ export class ImpactService extends BaseImpactService {
       entities,
     );
 
+    // If chart is requested for regions and specific level is received, we get only the requested level from admin regions trees of the response
+
+    if (
+      rankedImpactTableDto.groupBy === 'region' &&
```

Comment:
Following https://github.com/Vizzuality/landgriffon/pull/873/files#r1109319576, I still dont get why this is somehow tied to regions.

Does not make sense to apply this behaviour to any other entity that is a tree structure?